code/modules/events/destructible_terrain.dm

```diff
@@ -0,0 +1,11 @@
+//Enables destructible terrain on the ground level
+/proc/enable_destructible_terrain()
+	var/list/turfs = get_all_of_type(/turf/closed/wall, TRUE)
+	for(var/turf/closed/wall/turf_for_editing in turfs)
+		if(turf_for_editing.z != 3)
```

Comment:
is_ground_level() is a more robust check for this, but could be a lot more intensive

Reply:
Fixed